### ISSUE TITLE
[Feature Request]: Hide the CBMicroblocks from the EMI search list by default

### ISSUE THREAD

**Issue by antrobot1234** [NyagiByte/Compression#75]
Currently, the EMI search list is moderately bloated by the CBMicroblocks mod. It currently takes up 18 pages of the total 79 of the pack. I suggest removing all of them from the EMI search index to help reduce some of the clutter.
![image](https://github.com/user-attachments/assets/f6f3d75a-b96c-42c2-a235-880b014408ff)

**Comment by NyagiByte** [comment]
good call, ima put this on my to-do tho it wont be super high priority
